src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -322,275 +289,159 @@ const formConfig = {
       title: 'Healthcare information',
       pages: {
         hasPrimaryHealthInsurance: {
-          path: ':index/has-primary',
-          arrayPath: 'applicants',
-          showPagePerItem: true,
-          title: item => `${applicantWording(item)} primary health insurance`,
-          CustomPage: ApplicantHasPrimaryPage,
-          CustomPageReview: ApplicantHasPrimaryReviewPage,
+          path: 'has-primary',
+          title: formData =>
+            `${nameWording(formData)} primary health insurance`,
```

Comment:
This is the majority of changes in this file - removing list-loop boilerplate and updating the `title` property to use a new `nameWording` function.